go/test/endtoend/cluster/vttablet_process.go

```diff
@@ -295,20 +295,10 @@ func (vttablet *VttabletProcess) CreateDB(keyspace string) error {
 
 // QueryTablet lets you execute a query in this tablet and get the result
 func (vttablet *VttabletProcess) QueryTablet(query string, keyspace string, useDb bool) (*sqltypes.Result, error) {
-	dbParams := mysql.ConnParams{
-		Uname: "vt_dba",
-	}
-	if vttablet.DbPort > 0 {
-		dbParams.Port = vttablet.DbPort
-	} else {
-		dbParams.UnixSocket = path.Join(vttablet.Directory, "mysql.sock")
-	}
-	if useDb {
-		dbParams.DbName = "vt_" + keyspace
-	}
-	if vttablet.DbPassword != "" {
-		dbParams.Pass = vttablet.DbPassword
+	if !useDb {
+		keyspace = ""
 	}
+	dbParams := NewConnParams(vttablet.DbPort, vttablet.DbPassword, path.Join(vttablet.Directory, "mysql.sock"), keyspace)
```

Comment:
I don't see the use of abstracting the parameter setting to a new method. Is that used anywhere else?

Reply:
this method we use in vttablet_process.go and this file